api/types/access_request.go

```diff
@@ -401,6 +401,11 @@ func (r *AccessRequestV3) SetLoginHint(login string) {
 	r.Spec.LoginHint = login
 }
 
+// SetAssumedRoleRequestID sets the request ID of the access request from which the assumed role was obtained
+func (m *Metadata) SetAssumedRoleRequestID(requestId string) {
```

Comment:
Isn't `Metadata` used for all resources? You only need this request ID on the session resource, right?

I would put this in the WebSessionV2Spec